worldtimebuddy/cli.py

```diff
@@ -4,12 +4,28 @@
 
 MAJOR_TIMEZONES = ['UTC', 'US/Pacific', 'Asia/Kolkata']
 
+# Mapping of short timezone codes to full names
+timezone_codes = {
+    'IST': 'Asia/Kolkata',
+    'PST': 'US/Pacific',
+    'EST': 'US/Eastern',
+    'CST': 'US/Central',
+    'MST': 'US/Mountain',
+    'GMT': 'Europe/London',
+    'CET': 'Europe/Paris',
+    'EET': 'Europe/Athens',
+    'JST': 'Asia/Tokyo',
+    'AEST': 'Australia/Sydney',
+    'UTC': 'UTC'
+}
```

Comment:
Consider this adding a separate constants file.

Reply:
will get back to it right away

Reply:
Also please add testing details for both the changes.

Reply:
> Also please add testing details for both the changes.

would screen shots be enough?

Reply:
Can you check the PR - https://github.com/dubesar/worldtimebuddy/pull/7, attach similarly with different cases.